format/FormatMisc.js

```diff
@@ -42,6 +42,21 @@ export function fmtSpan(v, {
     return asHtml ? fmtSpanHtml(v, opts) : fmtSpanElement(v, opts);
 }
 
+/**
+ * Format JSON string
+ *
+ * @param {string} str - string to be formatted into JSON
```

Comment:
Ideally we would call this something like `prettyPrintJson()` but we have a strong convention of the leading `fmt` on these formatter function names.

We could clarify in the comment:
```
**
 * Pretty-print a JSON string, adding line breaks and indentation.
 *
 * @param {string} str - string to be pretty-printed, must be valid JSON.
 * ....
```

Reply:
Agreed.  jsDocs fixed.